client/src/components/PageExplorer/PageExplorer.scss

```diff
@@ -61,10 +61,14 @@ $explorer-header-horizontal-padding: 10px;
 .c-page-explorer__header__title__inner {
   @apply w-flex;
   padding: 1em $explorer-header-horizontal-padding;
+  padding-bottom: 1rem;
   overflow: hidden;
   text-overflow: ellipsis;
   white-space: nowrap;
 
+  outline: 1px solid #24184d;
+  outline-offset: -3px;
```

Comment:
```suggestion
```

This is also not needed - there is actually already a negative outline offset added via the mixin `show-focus-outline-inside` in this file.

Reply:
@lb- learnt a lot of things from this pull request. Thank you for guiding me, It's been really a great experience.

Reply:
no problems - thanks for putting your code out there for feedback - it is always scary to do that but well done.
I hope these first two PRs help you get a feel for the process.